test/unit/TokenController/totalBalanceOf.js

```diff
@@ -0,0 +1,155 @@
+const { expect } = require('chai');
+const { ethers } = require('hardhat');
+const { setEtherBalance } = require('../../utils/evm');
+const { One, MaxUint256 } = ethers.constants;
+
+const poolId = 234;
+describe('totalBalanceOf', function () {
+  beforeEach(async function () {
+    const { nxm } = this.contracts;
+    const [member1] = this.accounts.members;
```

Comment:
Since it's a single member that you're using - name it `member` instead of `member1`.